cypress/e2e/automatically-focus-on-editor.cy.js

```diff
@@ -0,0 +1,23 @@
+describe("it will check if the focus is on editor on desktop and not in mobiles", () => {
+    context("720p resolution", () => {
+        beforeEach(() => {
+            cy.viewport(1280, 720);
+            cy.visit("/");
+        });
```

Comment:
Since there's not much code, can we remove the `beforeEach` and simply extract the code outside? I.e.
instead of
```
        beforeEach(() => {
            cy.viewport(1280, 720);
            cy.visit("/");
        });
```
we can just have
```
cy.viewport(1280, 720);
            cy.visit("/");
```
? In both cases.